packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -129,22 +134,22 @@ class ImageCache {
   ImageStreamCompleter putIfAbsent(Object key, ImageStreamCompleter loader()) {
     assert(key != null);
     assert(loader != null);
-    ImageStreamCompleter result = _pendingImages[key];
+    _PendingImage result = _pendingImages[key];
```

Comment:
since you don't actually return `result` anymore, we shouldn't call it `result`.